replica/replica.go

```diff
@@ -606,37 +612,19 @@ func (r *Replica) processPrepareRemoveDisks(disks []string) ([]PrepareRemoveActi
 			return nil, fmt.Errorf("Wrong disk %v doesn't exist", disk)
 		}
 
-		children := r.diskChildrenMap[disk]
-		// 1) leaf node
-		if children == nil {
-			actions = append(actions, PrepareRemoveAction{
+		parent := r.diskData[disk].Parent
+
+		actions = append(actions,
+			PrepareRemoveAction{
+				Action: OpCoalesce,
+				Source: disk,
```

Comment:
is this way of coalescing supported by the 'sync' binary?
I remember that there is a reason for copying from a disk to its child, rather than copying from a disk to its parent.

Reply:
We are always copying data from child to parent, since child has the latest data.